### DISCUSSION TITLE
Renovate v39 Docker image does not include npm anymore

### DISCUSSION THREAD

**Discussion by neophob** [renovatebot/renovate#32449] [Request Help] [answered]
### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

We use npm to configure an internal npm repo, that worked fine up to renovate v38 but fails in renovate v39 because npm is not included in the default docker image:

```
╰─❯ docker run -it --rm ghcr.io/renovatebot/renovate:38.142.7 bash
ubuntu@952830c14440:/usr/src/app$ node -v
v20.18.0
ubuntu@952830c14440:/usr/src/app$ npm -v
10.8.2
ubuntu@952830c14440:/usr/src/app$ exit

╰─❯ docker run -it --rm ghcr.io/renovatebot/renovate:39.9.4 bash
ubuntu@5a84b6e446a6:/usr/src/app$ node -v
v22.11.0
ubuntu@5a84b6e446a6:/usr/src/app$ npm -v
bash: npm: command not found
ubuntu@5a84b6e446a6:/usr/src/app$
```

are there specific reasons why npm was removed?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
You can use the `-full` image if you need package managers preinstalled.

```
$ docker run -it --rm ghcr.io/renovatebot/renovate:39.9.4-full bash
ubuntu@60e0475c967b:/usr/src/app$ node --version
v22.11.0
ubuntu@60e0475c967b:/usr/src/app$ npm --version
10.9.0
```
